app/renderer/js/utils/certificate-util.js

```diff
@@ -0,0 +1,91 @@
+'use strict';
+
+const { app, dialog } = require('electron').remote;
+const fs = require('fs');
+const path = require('path');
+const JsonDB = require('node-json-db');
+const Logger = require('./logger-util');
+
+const logger = new Logger({
+	file: `certificate-util.log`,
+	timestamp: true
+});
+
+let instance = null;
+
+class CertificateUtil {
+	constructor() {
+		if (instance) {
+			return instance;
+		} else {
+			instance = this;
+		}
+
+		this.reloadDB();
+		return instance;
+	}
+	getCertificate(server, defaultValue = null) {
+		this.reloadDB();
+		const value = this.db.getData('/')[server];
+		if (value === undefined) {
+			return defaultValue;
+		} else {
+			return value;
+		}
+	}
+	// Function to copy the certificate to userData folder
+	copyCertificate(server, location, fileName) {
+		let copied = false;
+		const filePath = this.generateFilePath(fileName);
+		try {
+			fs.copyFileSync(location, filePath);
+			copied = true;
+		} catch (err) {
+			console.log(err);
+			dialog.showErrorBox(
+				'Error saving certificate',
+				'We encountered error while saving the certificate.'
+			);
+		}
+		return copied;
+	}
+	setCertificate(server, fileName) {
+		const filePath = this.generateFilePath(fileName);
+		this.db.push(`/${server}`, filePath, true);
+		this.reloadDB();
+	}
+	removeCertificate(server) {
+		this.db.delete(`/${server}`);
+		this.reloadDB();
+	}
+	generateFilePath(filename) {
+		const dir = `${app.getPath('userData')}/certificates`;
+
+		// Create 'certificates' directory if not existed
+		if (!fs.existsSync(dir)) {
+			fs.mkdirSync(dir);
+		}
+
+		return `${dir}/${filename}`;
+	}
+	reloadDB() {
```

Comment:
This exactly looks like the one which we have in the config-util.

Reply:
Yes, it's the same template that I've followed. Modified it for `certifcates.json`. Since we need to reload it after every access. Right?

Reply:
I think it's okay for now but we should find a better way to reload the db. We can make a function which basically takes a db file as an argument and then we can reuse and pass it in other utils also.

Reply:
Yes that is good idea. Maybe add it in something like `db-util.js`  later.